Products/CMFEditions/StandardModifiers.py

```diff
@@ -28,6 +28,10 @@
 import os,sys
 from itertools import izip
 from App.class_init import InitializeClass
+from plone.dexterity.interfaces import IDexterityContent
```

Comment:
I'm not sure how I feel about P.CMFEditions depending on plone.dexterity... aren't we introducing a circular dependency here?